packages/Card/handleResponsiveStyles.js

```diff
@@ -0,0 +1,37 @@
+import { prepareArray, generateStyles } from '@tds/util-helpers'
+
+const MOBILE_BREAKPOINTS = ['xs', 'sm']
+const DESKTOP_BREAKPOINTS = ['md', 'lg', 'xl']
+
+export const isMobileBreakpoint = breakpoint => MOBILE_BREAKPOINTS.indexOf(breakpoint) !== -1
+export const isDesktopBreakpoint = breakpoint => DESKTOP_BREAKPOINTS.indexOf(breakpoint) !== -1
+
+const handleBoundaryCrossing = (acc, curr) => {
+  if (
+    isMobileBreakpoint(curr.from) &&
+    ((curr.until !== 'md' && isDesktopBreakpoint(curr.until)) || typeof curr.until === 'undefined')
+  ) {
+    const props = Object.keys(curr.props).filter(
+      prop => typeof curr.props[prop] === 'number' && curr.props[prop] > 3
+    )
+    if (props.length !== 0) {
+      const mobileBreakpoint = { ...curr, props: curr.props }
+      const desktopBreakpoint = { ...curr, props: curr.props }
+
+      mobileBreakpoint.until = 'md'
+      desktopBreakpoint.from = 'md'
+
+      return acc.concat([mobileBreakpoint, desktopBreakpoint])
+    }
+  }
+  return acc.concat([curr])
+}
+
+const handleResponsiveStyles = (props, styleFn) => {
+  const breakpoints = prepareArray(props)
+    .filter(bp => Object.keys(bp.props).length > 0)
+    .reduce(handleBoundaryCrossing, [])
+  return generateStyles(breakpoints, styleFn)
+}
+
+export default handleResponsiveStyles
```

Comment:
I see that you copied this function `handleResponsiveStyles` from the Box component. It's better to define a function once and reuse it in multiple places than copy and paste it each time you want to use it.

That way, you end up having less code to maintain. For more info [read here](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

With that spirit, I suggest you move this function to `'@tds/util-helpers'` package and import it to both Card and Box components.